### DISCUSSION TITLE
Lab 27 Not Currently On Branch

### DISCUSSION THREAD

**Discussion by kylelin23** [hack4impact-calpoly/starter-pack#13] [Q&A] [answered]
Hey, I was working on lab 27 and ran into an error that prevented me from connecting Github. 
I did the command $ git remote -v and got back: 
<img width="480" alt="Screenshot 2023-10-02 at 6 36 21 PM" src="https://github.com/hack4impact-calpoly/starter-pack/assets/85212405/e4949ca1-04a2-431f-82ac-535d1de76552">

This led me to believe that my Github should be connected all good, but when I ran the command $ git branch -M main, I got: 
<img width="481" alt="Screenshot 2023-10-02 at 6 39 56 PM" src="https://github.com/hack4impact-calpoly/starter-pack/assets/85212405/eb422c44-1e51-4d28-a508-63d542ea8f63">

This made my git push command not work: 
<img width="585" alt="Screenshot 2023-10-02 at 6 42 45 PM" src="https://github.com/hack4impact-calpoly/starter-pack/assets/85212405/22fdb951-c5f4-4ca7-a583-e63ed730d4f0">

Do you know how I can try to get myself onto the branch? I tried some advice from online such as $ git pull, but none of it worked. After doing git pull, I got this: 
<img width="460" alt="Screenshot 2023-10-02 at 6 44 03 PM" src="https://github.com/hack4impact-calpoly/starter-pack/assets/85212405/ec31f123-1e2c-44db-ac8f-1a32df35241d">

Thank you, 
Kyle

**Comment by kylelin23** [comment, marked as answer]
I figured the problem out, git checkout main returned me onto the main branch.

**Reply by khoa-l** [reply]
Glad you figured that out! Sorry, this slipped my mind 😓
